components/02-elements/swatch/swatch.config.js

```diff
@@ -0,0 +1,66 @@
+module.exports = {
+  context: {
+    class: '',
+    attributes: '',
+    titleClass: '',
+    title: 'Choose your size',
+    options: [
+      {
+        class: '',
+        attributes: '',
+        text: 'XS'
+      },
+      {
+        class: '',
+        attributes: '',
+        text: 'S'
+      },
+      {
+        class: '',
+        attributes: '',
+        text: 'M'
+      },
+      {
+        class: '',
+        attributes: '',
+        text: 'L'
+      },
+      {
+        class: '',
+        attributes: '',
+        text: 'XL'
+      }
+    ]
+  },
+  variants: [
+    {
+      name: 'color',
+      context: {
+        class: 'swatch--color',
+        title: 'Choose color',
+        options: [
+          {
+            class: 'swatch__option--color',
```

Comment:
This class doesn't exist in the stylesheet.

Reply:
removed